Makefile

```diff
@@ -145,6 +145,8 @@ standalone: node_modules
 .PHONY: add-on
 add-on: node_modules
 	mkdir -p $(BUILT)/$@
+	sed -n -e '/^%/p' $@/jar.mn | sed -e "s/^% //g" | sed -e "s/%/chrome\//g" > $@/chrome.manifest
+	sed -e '/^%/d' $@/jar.mn > $(BUILT)/$@/jar.mn
```

Comment:
We don't need to generate the jar.mn here, that's just the source (we need it for when we export back to m-c).